Sources/SwiftParserDiagnostics/MissingNodesError.swift

```diff
@@ -364,7 +370,10 @@ extension ParseDiagnosticsGenerator {
           missingNodes += [sibling]
         } else if sibling.isMissingAllTokens && sibling.hasTokens {
           missingNodes += [sibling]
-        } else if sibling.kind.isSyntaxCollection && sibling.children(viewMode: .sourceAccurate).count == 0 {
+        } else if sibling.kind.isSyntaxCollection
+          && (sibling.children(viewMode: .sourceAccurate).count == 0
+            || sibling.children(viewMode: .sourceAccurate).only?.kind == .stringSegment)
```

Comment:
(If my approach is correct...) How about separating this part as a property named `isSingleToken`(or a more appropriate name) and accessing it as `sibling.isSingleToken`?